webui/react/src/pages/Settings/UserManagement.tsx

```diff
@@ -155,33 +154,26 @@ const UserManagement: React.FC = () => {
   const [groups, setGroups] = useState<V1GroupSearchResult[]>([]);
   const pageRef = useRef<HTMLElement>(null);
   const { settings, updateSettings } = useSettings<UserManagementSettings>(settingsConfig);
-  const params: GetUsersParams = useMemo(
-    () => ({
-      limit: settings.tableLimit,
-      name: settings.name,
-      offset: settings.tableOffset,
-      orderBy: settings.sortDesc ? 'ORDER_BY_DESC' : 'ORDER_BY_ASC',
-      sortBy: validateDetApiEnum(V1GetUsersRequestSortBy, settings.sortKey),
-    }),
-    [settings],
-  );
-  const loadableUsers = useObservable(userStore.getUsers(params));
+
+  const loadableUsers = useObservable(userStore.getUsers());
   const users = Loadable.getOrElse([], loadableUsers);
-  const pagination = Loadable.getOrElse(
-    undefined,
-    useObservable(userStore.getUsersPagination(params)),
-  );
-  const canceler = useRef(new AbortController());
+
+  const filteredUsers = settings.name
+    ? users.filter((user) => {
+        return (user.displayName || user.username).includes(settings.name ?? '');
```

Comment:
nit: maybe `settings.name && (user.displayName || user.username).includes(settings.name)`? 
in case  username could be `''` (this shouldnt happen but just in case...).

Reply:
makes sense to me, avoid having to do a search if search string is empty